docs/articles/vs-test-adapter/NUnit-And-Microsoft-Test-Platform.md

```diff
@@ -0,0 +1,55 @@
+# NUnit and Microsoft Test Platform
+
+## Overview
+
+Microsoft Test Platform (MTP) is the new platform for testing from Microsoft.  The Test Projects can be run as 
+executables, like we do with [NUnitLite](../nunit/running-tests/NUnitLite-Runner.md). There is no longer a test runner,
+the executable **is** the test runner. 
```

Comment:
```suggestion
the executable _is_ the test runner. 
```

Reply:
You need to resuggest, it was overwritten